starter-code/server.js

```diff
@@ -30,7 +30,7 @@ function proxyGitHub(request, response) {
 
 
 // COMMENT: What is this route doing? Where does it receive a request from?
-// (put your response in a comment here)
+// (put your response in a comment here) Whenever there is a get request on a root path we send the data in callback funtion.
```

Comment:
when /new route is accessed a get request is sent to your root directory that returns new.html